test/RewardsStreamerMP.t.sol

```diff
@@ -2954,21 +2995,22 @@ contract FuzzTests is StakeManagerTest {
     }
 
     function _expectUnstake(address account, uint256 amount) internal {
+        CheckVaultParams storage expectedAccountParams = expectedAccountState[account];
+        expectedAccountParams.account = vaults[account];
+        if (expectedVaultLockState[expectedAccountParams.account].lockEnd >= vm.getBlockTimestamp()) {
+            expectedRevert = IStakeManager.StakeManager__FundsLocked.selector;
+            console.log("locked");
```

Comment:
do we want to have this log or did you forgot it?

Reply:
This is just moved code. I can remove it while I'm at it.